apps/theming-designer/src/components/ThemingDesigner.tsx

```diff
@@ -69,14 +69,20 @@ const Main = (props: IStackProps) => (
 );
 
 let colorChangeTimeout: number;
+let fabricPaletteColorChangeTimeout: number;
 
 export class ThemingDesigner extends BaseComponent<{}, IThemingDesignerState> {
-  constructor(props: any) {
+  constructor(props: {}) {
     super(props);
 
     this.state = this._buildInitialState();
   }
 
+  public componentWillUnmount() {
+    clearTimeout(colorChangeTimeout);
```

Comment:
since you use this.__async, i think the timeouts are automatically cleared when the component unmounts..